### ISSUE TITLE
Can't get this to work :(

### ISSUE THREAD

**Issue by Marketingguru** [coreytrampe/shopify-exfil#1]
it says fetching data from shopifystore.com but doesnt do anything after that... I got all the dependencies but still don't know what im doing what :(

**Comment by coreytrampe** [comment]
Is that the URL that you gave it? Try using a X.myshopify.com URL.

**Comment by Marketingguru** [comment]
No I was using an actual shopify url haha, it just says fetching data from
example.myshopify.com and then doesn't do anything. I run, node index.js
example.myshopify.com

**Comment by coreytrampe** [comment]
Ok, you can pull and try it again. It wasn't working with http protocol all of a sudden, and I'm guessing Shopify started redirecting http to https since I tested it last.

I was able to use it with https://sample-store-57.myshopify.com. I mean, I got a CSV. I didn't try to import it.

When I tested on a store with a much larger catalog, it looked like it ran into a rate limiting issue that also appears to be new, and it does not handle it at all. I guess you can file a separate issue if you run into that.

**Comment by Marketingguru** [comment]
Yeah I ran into that haha, I was able to export 4200 products from a store that I tested with a different script, but yours seems to grab more data from each product. It looks like if shopify blocks you from sending too many requests, it doesn't export all at. How could I limit it to a specific amount or make it wait like 10 minutes and continue.
